### PR TITLE
Docs: Update alerting notifications documentation 

### PR DESCRIPTION
**Pull request by yuri-tceretian**
**What is this feature?**
This PR aggregates many changes we've made in Grafana alerting role-based access control. Currently, all the changes are behind the feature flag `alertingApiServer`.

### CODE REVIEW

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 1]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
 1. Hover your mouse cursor over a folder and click **Go to folder**.
 1. Click **Manage permissions** from the Folder actions menu.
 1. Update or add permissions as required.
+
+## Manage access using contact point permissions
+
+Available only when feature flag `alertingApiServer` is enabled.
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:70]:
```suggestion

```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 2]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+
+You can extend or limit the access provided by a role to contact points by assigning permissions to individual contact point.
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:72]:
```suggestion
Extend or limit the access provided by a role to contact points by assigning permissions to individual contact point.
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 3]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+
+This allows different users, teams, or service accounts to have customized access to read or modify specific contact points.
+
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:75]: ## Before you begin

- Enable the `alertingApiServer` feature toggle.

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 4]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+Refer to the following table for details on the additional access provided by contact point permissions:
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:76]:
```suggestion
Refer to the following table for details on the additional access provided by contact point permissions.
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 5]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+
+| Folder permission | Additional Access                                                                                                                            |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| View              | Ability to view and export contact point as well as select it in Alert rule edit page                                                        |
+| Edit              | Update or delete the contact point                                                                                                           |
+| Admin             | Same additional access as Edit and manage permissions for the contact point. User should have additional permissions to read users and teams |
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:82]:
```suggestion
| Admin             | Same additional access as Edit and manage permissions for the contact point. User should have additional permissions to read users and teams. |
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 6]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+You can extend or limit the access provided by a role to contact points by assigning permissions to individual contact point.
+
+This allows different users, teams, or service accounts to have customized access to read or modify specific contact points.
+
+Refer to the following table for details on the additional access provided by contact point permissions:
+
+| Folder permission | Additional Access                                                                                                                            |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| View              | Ability to view and export contact point as well as select it in Alert rule edit page                                                        |
+| Edit              | Update or delete the contact point                                                                                                           |
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:81]:
```suggestion
| View              | Ability to view and export contact point as well as select it in Alert rule edit page.                                                        |
| Edit              | Update or delete the contact point.                                                                                                           |
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 7]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+| Admin             | Same additional access as Edit and manage permissions for the contact point. User should have additional permissions to read users and teams |
+
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:83]: ### Steps

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 8]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+To contact point permissions, complete the following steps.
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:84]:
```suggestion
To update contact point permissions, complete the following steps.
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 9]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:85]: this is only for admins right? or can other users change permissions? isn't there some rule that you can edit if you created the contact point, too?

File: docs/sources/alerting/set-up/configure-roles/index.md (hunk 2 above)
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:72]:
```suggestion
You can extend or limit the access provided by a role to contact points by assigning permissions to an individual contact point.
```

File: docs/sources/alerting/set-up/configure-roles/index.md [hunk 10]
```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
...
+You can extend or limit the access provided by a role to contact points by assigning permissions to individual contact point.
+
+This allows different users, teams, or service accounts to have customized access to read or modify specific contact points.
+
+Refer to the following table for details on the additional access provided by contact point permissions:
+
+| Folder permission | Additional Access                                                                                                                            |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| View              | Ability to view and export contact point as well as select it in Alert rule edit page                                                        |
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-roles/index.md:80]:
```suggestion
| View              | View and export contact point as well as select it on the Alert rule edit page                                                        |
```

File: docs/sources/alerting/set-up/configure-roles/index.md (hunk 9 above)
Comment (yuri-tceretian) [reply docs/sources/alerting/set-up/configure-roles/index.md:85]: This is for all users who can administer a contact point. Basic "Admin" role can do that for all. User that creates a new contact point is automatically granted admin permissions to it, and therefore that user can grant admin permissions to other users\teams

**Review by jdbaldry — COMMENTED** [review]
We should avoid using `<br>` tags to alter table presentation. Instead, prefer `<p>` or list tags to properly semantically mark up the document.

https://developers.google.com/style/tables#multi-paragraph-table-cells

There's an example in https://github.com/grafana/grafana/blob/main/docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md?plain=1#L43-L183

**Review by brendamuir — APPROVED** [review]

**Pull request merged**